dozer-types/Cargo.toml

```diff
@@ -38,6 +39,7 @@ tokio-postgres = { version = "0.7.7", features = [
 serde_bytes = "0.11.12"
 arbitrary = { version = "1", features = ["derive"], optional = true }
 schemars = "0.8.15"
+rmp-serde = "1.1.2"
```

Comment:
Why we're using message pack and bincode together?

Reply:
Bincode is not self-describing, which in turn means it can't express JSON. Practically, this means that bincode doesn't support serde's `deserialize_any()`, which JSON implementations need, as the data type of the serialized data is given by the actual contents of the serialized format.

In this case, switching to msgpack inside the bincode implementation was the easiest, as msgpack matches JSON quite closely, and is almost as fast and compact as bincode.